syndesis/src/modules/customizations/customizations.css

```diff
@@ -0,0 +1,21 @@
+.bottom-bordered {
```

Comment:
It would be better not to have CSS in the main app, unless we are talking about app-wide utility classes.

These look like are specific to the customization section, so the components using them should be refactored to be exported by the syndesis/ui package, and classes renamed to leverage the BEM methodology. The convention I have been following is `.{component-name}__{component-item}` (eg. `.content-with-sidebar__sidebar`).

Reply:
I created `NavLinkTab` in ui/Shared to address this comment. Wdyt?

Reply:
@riccardo-forina I forgot to delete the `customizations.css file`. I'll amend the last commit and let you know.